gpu/README.md

```diff
@@ -0,0 +1,193 @@
+# Agent Check: GPU
+
+## Overview
+
+This check monitors GPU devices and their utilization through the Datadog Agent.
+
+Supported vendors: NVIDIA.
+
+- Track utilization of GPU devices and retrieve performance and health metrics.
+- Monitor processes that are using GPU devices and their performance.
+
+## Setup
+
+### Installation
+
+The GPU check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
+
+The check also uses eBPF probes to assign GPU usage and performance metrics to processes. eBPF programs are loaded by the `system-probe` component.
+
+**Note**: The `system-probe` GPU component (which generates per-process metrics) requires Linux kernel 5.8 or later. Windows is not supported.
+
+### Configuration
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+Enabling the `gpu` integration requires `system-probe` to have the configuration option enabled.  Inside the `system-probe.yaml` configuration file, the following parameters must be set:
+
+```yaml
+gpu_monitoring:
+  enabled: true
+```
+
+The check in the agent configuration file is enabled by default whenever NVIDIA GPUs and their drivers are detected in the system. However, it can also be configured manually following these steps:
```

Comment:
```suggestion
The check in the Agent configuration file is enabled by default whenever NVIDIA GPUs and their drivers are detected in the system. However, it can also be configured manually following these steps:
```